src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -271,14 +279,12 @@ trait ClusterComponent extends LeoComponent {
       }
     }
 
```

Comment:
Moved down the set* methods below so get*/set*/etc. methods are together for easier finding.